### PR TITLE
Release notes v4.5

### PR DESCRIPTION
**Pull request by juskora**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | 
| Versions      | 
| Edition       | 

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -78,7 +78,7 @@
                         <ul>
                             <li><a href="templating/render_content/render_content/">Content rendering</a></li>
                             <li><a href="search/search/">Search</a></li>
-                            <li><a href="release_notes/ibexa_dxp_v4.4/">Releases</a></li>
+                            <li><a href="release_notes/ibexa_dxp_v4.5/">Releases</a></li>
```
Comment (DominikaK) [inline docs/index.md:81]:
```suggestion
                            <li><a href="release_notes/release_notes/">Releases</a></li>
```

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+![The order list screen](img/4.5_order_list.png "The order list screen")
+
+#### Payment
+
+With the Payment module, users can define and manage payment methods, as well as create and manage payments, search for payments and filter the results. 
+Users can also enable or disable payment methods, modify payment information, and cancel payments.
+
+![The payment methods screen](img/4.5_shipping_methods.png "The payment methods screen")
+
+For more information, see [Commerce](https://doc.ibexa.co/en/4.4/commerce/commerce/).
```
Comment (DominikaK) [inline docs/release_notes/ibexa_dxp_v4.5.md:41]: Replace with feature-specific link once the doc is done

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+### Changes to the Checkout package
+
+The Checkout package has been modified by adding shipping management capabilities that let users define and manage shipping methods of different types and their related costs, as well as create and manage shipments, search for shipments and filter search results. 
+Users can also enable or disable shipping methods, change status of shipments, and cancel shipments.
+
+![The shipping methods screen](img/4.5_shipping_methods.png "The shipping methods screen")
+
+### Translation comparison
+
+With this release, you can compare different versions of the translations of the Content item, including comparison between different languages.
```
Comment (DominikaK) [inline docs/release_notes/ibexa_dxp_v4.5.md:52]:
```suggestion
With this release, you can compare different versions of translations of a Content item, including comparison between different languages.
```
or
```suggestion
With this release, you can compare different versions or translations of a Content item, including comparison between different languages.
```
?

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+### Translation comparison
+
+With this release, you can compare different versions of the translations of the Content item, including comparison between different languages. 
+
+You can now choose between two new options of the view:
+
+- Split - default, side by side view to compare versions of the same or different languages
+- Unified - single column view to compare versions of the same language
+
+Now, when you compare different versions within the same language, the system highlights the changes using the colors:
```
Comment (DominikaK) [inline docs/release_notes/ibexa_dxp_v4.5.md:59]:
```suggestion
Now, when you compare different versions within the same language, the system highlights the changes using colors:
```

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+
+### New commerce page blocks [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
+
+This release introduces new page blocks:
+
+- [Bestsellers block](https://doc.ibexa.co/projects/userguide/en/4.5/content_management/block_reference/#bestsellers-block) displays a list of products from PIM that were recently a bestseller.
+
+![Bestsellers block](img/4.5_bestsellers_block.png "Bestsellers block")
+
+- [React app block](https://doc.ibexa.co/projects/userguide/en/4.5/content_management/pages/#react-app-block) allows an editor to embed a preconfigured React application into a page. React app block requires configuration. For more information, see [React App Block configuration](https://doc.ibexa.co/projects/userguide/en/4.5/content_management/pages/react_app_block/#react-app-block-configuration).
```
Comment (DominikaK) [inline docs/release_notes/ibexa_dxp_v4.5.md:77]:
```suggestion
- [React app block](https://doc.ibexa.co/projects/userguide/en/4.5/content_management/pages/#react-app-block) allows an editor to embed a preconfigured React application in a page. React app block requires configuration. For more information, see [React App Block configuration](https://doc.ibexa.co/projects/userguide/en/4.5/content_management/pages/react_app_block/#react-app-block-configuration).
```

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+
+![React app block](img/4.5_react_app_block.png "React app block")
+
+### Page Builder for B2B portalss [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
+
+With this release, you will be able to use Page Builder to create custom Customer Portals for your clients. 
+With new Sales rep page block and using all available blocks from original Page Builder, you can create unique experience for each customer group.
```
Comment (DominikaK) [inline docs/release_notes/ibexa_dxp_v4.5.md:84]:
```suggestion
With new Sales rep page block and using all available blocks from original Page Builder, you can create a unique experience for each customer group.
```

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 7]
```diff
@@ -0,0 +1,205 @@
...
+Additionally, you can assign each customer group to a specific Customer Portal or create an availability hierarchy based on rules and configuration.
+
+![Page Builder for B2B portals](img/4.5_page_builder_b2b.png "Page Builder for B2B portals")
+
+### Personalization improvements
+
+#### New B2B models in Personalization engine
+
+Personalization engine introduces two new types of models: [last clicked and last purchased B2B, and B2B recurring purchase models](https://doc.ibexa.co/projects/userguide/en/master/personalization/recommendation_models/#b2b-model), dedicated to B2B users.
+Built on the fly, and based on segment groups models return actual items clicked by user with the same segment ID and actual bought items.
```
Comment (DominikaK) [inline docs/release_notes/ibexa_dxp_v4.5.md:94]:
```suggestion
Built on the fly, and based on segment groups, the models return actual items clicked by users with the same segment ID and actual bought items.
```

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 8]
```diff
@@ -0,0 +1,205 @@
...
+B2B recurring purchase model anticipates and predicts purchase of products that were bought recursively within the same segment ID.
```
Comment (DominikaK) [inline docs/release_notes/ibexa_dxp_v4.5.md:95]: recursively or repeatedly?

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 9]
```diff
@@ -0,0 +1,205 @@
...
+
+### Segment management
+
+Now you can use segmentation logic with operators to can build complex segment groups which enable precise filtering.
```
Comment (DominikaK) [inline docs/release_notes/ibexa_dxp_v4.5.md:99]:
```suggestion
Now you can use segmentation logic with operators to build complex segment groups which enable precise filtering.
```

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 10]
```diff
@@ -0,0 +1,205 @@
...
+groups to get the most accurate, precise and targeted recommendations for your customers.
+
+![Segment management](img/4.5_segment_management.png "Segment management logic")
+
+## Other changes
+
+### Customer Data Platform (CDP) configuration
+ 
+In this release, the CDP configuration becomes more generic
+and allows us to support other transport types accepted by CDP.
```
Comment (DominikaK) [inline docs/release_notes/ibexa_dxp_v4.5.md:110]:
```suggestion
and allows supporting other transport types accepted by CDP.
```

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 11]
```diff
@@ -0,0 +1,205 @@
...
+Currently, only `stream_file` transport is supported and can be initialized from the configuration.
+
+Ibexa v4.5 adds the abstraction that allows you to implement other transport types from third parties.
```
Comment (DominikaK) [inline docs/release_notes/ibexa_dxp_v4.5.md:113]:
```suggestion
Ibexa DXP v4.5 adds the abstraction that allows you to implement other transport types from third parties.
```

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 12]
```diff
@@ -0,0 +1,205 @@
...
+For more information, see [CDP configuration](https://doc.ibexa.co/en/4.5/cdp/cdp_activation/#configuration).
+
+### API improvements
+
+#### REST API for company accounts [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
+
+This release, adds new endpoints that will allow you to manage companies in your platform with REST API:
```
Comment (DominikaK) [inline docs/release_notes/ibexa_dxp_v4.5.md:120]:
```suggestion
This release adds new endpoints that allow you to manage companies in your platform with REST API:
```

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 13]
```diff
@@ -0,0 +1,205 @@
...
+
+- GET `/corporate/companies` - supports pagination and existing Content Criteria and Sort Clauses but via query parameters
+- POST `/corporate/companies` - creates a company by using `CorporateAccount::createCompany` PAPI
```
Comment (DominikaK) [inline docs/release_notes/ibexa_dxp_v4.5.md:123]:
```suggestion
- POST `/corporate/companies` - creates a company
```
not really relevant which method lies underneath it

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 14]
```diff
@@ -0,0 +1,205 @@
...
+#### PHP API for payment methods and payments [[% include 'snippets/commerce_badge.md' %]]
+
+The Payment package provides the following services that are entrypoints to the backend API:
+
+- `Ibexa\Contracts\Payment\PaymentServiceInterface` for managing payments
+- `Ibexa\Contracts\Payment\PaymentMethodServiceInterface` for managing payment methods
+
+### Category filter in product search
+
+To help users search for products, products in Main Catalog view can now be filtered by product category.
```
Comment (DominikaK) [inline docs/release_notes/ibexa_dxp_v4.5.md:170]:
```suggestion
To help users search for products, products in the main catalog view can now be filtered by product category.
```

**Review by DominikaK — CHANGES_REQUESTED** [review]

File: docs/release_notes/ibexa_dxp_v4.5.md (hunk 8 above)
Comment (juskora) [reply docs/release_notes/ibexa_dxp_v4.5.md:95]: Recursively.

**Review by DominikaK — APPROVED** [review]

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 15]
```diff
@@ -0,0 +1,208 @@
...
+- POST `/orders/orders/{id}` - cancels an order
+- PATCH `/orders/orders/{id}` - updates an order
+
+#### PHP API for order management [[% include 'snippets/commerce_badge.md' %]]
+
+The Order Management package provides the `Ibexa\Contracts\OrderManagement\OrderServiceInterface` service, which is the entrypoint for calling the backend API for managing orders.
+
+#### PHP API for shipping methods and shipments [[% include 'snippets/commerce_badge.md' %]]
+
+The Shipping package provides the following services that are entrypoints to the backend API:
```
Comment (dabrt) [inline docs/release_notes/ibexa_dxp_v4.5.md:159]:
```suggestion
The Checkout package provides the following services that are entrypoints to the backend API:
```

**Review by radzyu — CHANGES_REQUESTED** [review]
I've added my remarks and suggestions.

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 16]
```diff
@@ -0,0 +1,215 @@
...
+as well as create and manage payments, search for payments and filter the results. 
+Users can also enable or disable payment methods, modify payment information, and cancel payments.
+
+![The payment methods screen](img/4.5_shipping_methods.png "The payment methods screen")
+
+For more information, see [Commerce](https://doc.ibexa.co/en/4.4/commerce/commerce/).
+
+### Changes to the Checkout package
+
+The Checkout package has been modified by adding shipping management capabilities that let users
```
Comment (SylvainGuittard) [inline docs/release_notes/ibexa_dxp_v4.5.md:46]: Checkout also includes new payment methods.

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 17]
```diff
@@ -0,0 +1,215 @@
...
+as well as create and manage shipments, search for shipments and filter search results.
+Users can also enable or disable shipping methods, change status of shipments, and cancel shipments.
+
+![The shipping methods screen](img/4.5_shipping_methods.png "The shipping methods screen")
+
+### New commerce page blocks [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
+
+This release introduces new page blocks:
+
+- [Bestsellers block](https://doc.ibexa.co/projects/userguide/en/4.5/content_management/block_reference/#bestsellers-block) displays a list of products from PIM that were recently a bestseller.
```
Comment (SylvainGuittard) [inline docs/release_notes/ibexa_dxp_v4.5.md:57]: Should we mention that it requires Ibexa Personalization?

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 18]
```diff
@@ -0,0 +1,215 @@
...
+
+![Segment management](img/4.5_segment_management.png "Segment management logic")
+
+## Other changes
+
+### Customer Data Platform (CDP) configuration
+ 
+In this release, the CDP configuration becomes more generic
+and allows supporting other transport types accepted by CDP.
+Currently, only `stream_file` transport is supported and can be initialized from the configuration.
```
Comment (SylvainGuittard) [inline docs/release_notes/ibexa_dxp_v4.5.md:114]: Not sure to understand this paragraph. 
"allows supporting other transport types" and "only `stream_file` transport is supported" is kind of the opposite.

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 19]
```diff
@@ -0,0 +1,215 @@
...
+`\Ibexa\Contracts\CorporateAccount\Service\CorporateAccountService::createCompany` service instead of
+`\Ibexa\Contracts\CorporateAccount\Service\CompanyService::createCompany`.
+
+#### REST API for order management [[% include 'snippets/commerce_badge.md' %]]
+
+This release adds new endpoints that allow you to manage orders by using REST API:
+
+- GET `/orders/orders` - loads a list of orders
+- POST `/orders/orders` - creates an order 
+- GET `/orders/order` - loads an order by identifier
```
Comment (SylvainGuittard) [inline docs/release_notes/ibexa_dxp_v4.5.md:148]: "by identifier": which one?

File: docs/release_notes/ibexa_dxp_v4.5.md (hunk 16 above)
Comment (ViniTou) [reply docs/release_notes/ibexa_dxp_v4.5.md:46]: Checkout as a proces, but not package - it is in ibexa/payment

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 20]
```diff
@@ -0,0 +1,215 @@
...
+- `ibexa/payment`
+
+Modules can interact with each other, for example, to decrease stock as a result of a sale, or cancel shipments and payments when orders are cancelled, and so on.
+
+#### Order management
+
+With order management in place, users can now search for orders and filter search results. 
+They can also review order details and completion status, as well as cancel orders.
+
+![The order list screen](img/4.5_order_list.png "The order list screen")
```
[the remark this answers is not in the document]
Comment (dabrt) [reply docs/release_notes/ibexa_dxp_v4.5.md:32]: This was merely a placeholder

File: docs/release_notes/ibexa_dxp_v4.5.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (dabrt) [reply docs/release_notes/ibexa_dxp_v4.5.md:46]: It wasn't true until yesterday, now it is time to update the release notes.

File: docs/release_notes/ibexa_dxp_v4.5.md (hunk 17 above)
Comment (dabrt) [reply docs/release_notes/ibexa_dxp_v4.5.md:57]: Block description mentions the "Personalization scenario" field, so my guess is - no, there's no such need

File: docs/release_notes/ibexa_dxp_v4.5.md [hunk 21]
```diff
@@ -0,0 +1,219 @@
...
+
+### New commerce page blocks [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
+
+This release introduces new page blocks:
+
+- [Bestsellers block](https://doc.ibexa.co/projects/userguide/en/4.5/content_management/block_reference/#bestsellers-block) displays a list of products from PIM that were recently a bestseller.
+
+![Bestsellers block](img/4.5_bestsellers_block.png "Bestsellers block")
+
+- [React app block](https://doc.ibexa.co/projects/userguide/en/4.5/content_management/pages/#react-app-block) allows an editor to embed a preconfigured React application in a page. React app block requires configuration. For more information, see [React App Block configuration](https://doc.ibexa.co/projects/userguide/en/4.5/content_management/pages/react_app_block/#react-app-block-configuration).
```
Comment (dabrt) [inline docs/release_notes/ibexa_dxp_v4.5.md:61]: Both links seem to be broken.

**Pull request merged**